tests/unit/test_certificates_integration.py

```diff
@@ -12,24 +12,28 @@
 
 import tls_relation
 
+from .conftest import GATEWAY_CLASS_CONFIG, TEST_EXTERNAL_HOSTNAME_CONFIG
+
 
-@pytest.mark.usefixtures("patch_load_incluster_config")
 def test_generate_password(harness: Harness):
     """
     arrange: given a charm with no connectable container.
     act: when agent relation joined event is fired.
     assert: the event is deferred.
     """
     harness.begin()
+
     tls_rel = tls_relation.TLSRelationService(harness.charm.model)
+
     password = tls_rel.generate_password()
     assert isinstance(password, str)
     assert len(password) == 12
 
 
-@pytest.mark.usefixtures("patch_load_incluster_config")
 def test_cert_relation(
-    harness: Harness, certificates_relation_data: Dict[str, str], monkeypatch: pytest.MonkeyPatch
+    harness: Harness,
+    certificates_relation_data: Dict[str, str],
+    monkeypatch: pytest.MonkeyPatch,
 ):
     """
     arrange: given a charm with no connectable container.
```

Comment:
for this docstring too :D

Reply:
yep, I updated the docstring for this one too.